src/Common/src/System/Dynamic/Utils/DelegateHelpers.cs

```diff
@@ -10,6 +10,10 @@
 using System.Runtime.CompilerServices;
 using System.Reflection;
 
+#if FEATURE_REFEMIT
```

Comment:
What project sets this?  I don't see FEATURE_REFEMIT in any existing .cs or .csproj file in the repo.